SparkPost/2web2ui
# EO-743 V2 dashboards default to 25 per page

### What Changed
- V2 dashboard tables now default to 25 instead of 10

### How To Test
- [Point your local upstreams to staging or production](https://confluence.int.messagesystems.com/display/ENG/Configuring+Openresty+Upstreams)
- Log into appteam or account 108 for data
- Verify v2 tables show 25 entries (you might need to select a facet if the account does not have a lot of subaccounts)
- Verify v1 overview page show 10 entries

### To Do
